tree/30_generic_methods/service_stop.cf

```diff
@@ -39,14 +39,24 @@ bundle agent service_stop(service_name)
     "class_prefix"      string => canonify(join("_", "promisers"));
     "args"                     slist => { "${service_name}" };
 
+  class:
+
+    "is_inner_method" expression => "!${report_data.should_report}";
```

Comment:
Shouldn't we do that on ever method ? since any method can become a inner method

Reply:
Maybe, I suggest to do likewise in another PR :)

Reply:
ok, in this case, this PR is OK for me